CMakeLists.txt

```
@@ -557,6 +557,7 @@ add_custom_command(
     -DOUT_FILE=${PROJECT_BINARY_DIR}/include/generated/version.h
     -DVERSION_TYPE=KERNEL
     -DVERSION_FILE=${ZEPHYR_BASE}/VERSION
+    -DKERNEL_VERSION_CUSTOMIZATION="${KERNEL_VERSION_CUSTOMIZATION}"
```

How is this supposed to work? Should I be able to pass:
```
west build . -DKERNEL_VERSION_CUSTOMIZATION="#define BANNER_VERSION \\\"blah\\\""
```
If so, this should be wrapped to support spaces:
```suggestion
    "-DKERNEL_VERSION_CUSTOMIZATION=${KERNEL_VERSION_CUSTOMIZATION}"
```

---

the intention is that this can be specified in a CMake file, but it is possible to use command line.

Just note you must pass a CMake ;-list, because the `;` transforms into spaces upon expansion.
For example like this:
```
-DKERNEL_VERSION_CUSTOMIZATION="#define;BANNER_VERSION;STRINGIFY(blah)"
```

So all qoutes are placed correctly 😄